src/content/smart-bulletins/034.md

```diff
@@ -0,0 +1,47 @@
+
+Since JPM SP2 contract performance period ended and there is no longer a valid Authorization to Operate (ATO) in place, JPM was required to take their SP2 Electronic Access System (EAS), also known as “PaymentNet,” offline, resulting in JPM customer agencies under SP2 no longer being able to access PaymentNet data after July 2019. In addition, the Federal Information Security Management Act (FISMA) and GSA’s information technology security requirements impose further restrictions which do not allow the SP2 data to reside in JPM’s system without a valid ATO. As a result, GSA and JPM subsequently agreed JPM would provide CCCM with relevent GSA SP2 data. Following the transmission of an SP2 data extract to GSA, JPM will move all SP2 data into long term storage which can only be accessed pursuant to legal and regulatory requirements. On May 6, 2026, JPM will purge all SP2 data from their systems. 
+
+Starting July 29, 2020 all requests for JPM SP2 data will be handled by CCCM. When requesting data from CCCM, agencies shall specify the following: 
+- Purpose of the data request
+- Reporting period (e.g., fiscal year(s), quarter(s), month(s), etc.)
+- Desired data elements
+- Preferred file type for delivery (options include: .xlsx/.xls, .csv, or .txt)
+
+When agencies submit proper data requests, CCCM will provide the requested information within 30 calendar days. Agencies can expect the data to be delivered in an email attachment (if the file size is 25MB or less) via the gsa_smartpay@gsa.gov email address. If the file size is larger than 25MB, CCCM will discuss with the requesting agency on alternate delivery method that would be acceptable to both parties. Please note that the data will be encrypted should it contain sensitive information. 
+
+## Action
+
+Agencies requiring JPM SP2 data should submit requests to CCCM through the following email: gsa_smartpay@gsa.gov.
```

Comment:
Agencies requiring JPM SP2 data should submit requests to CCCM through the following email: gsa_smartpay@gsa.gov.
Agencies should review their current card policies and guidance and update based on additional purchase card utilization opportunities available under the GSA MAS program.